Common/src/main/java/com/blamejared/crafttweaker/CraftTweakerCommon.java

```diff
@@ -60,6 +67,16 @@ public static void init() {
         }).start();
     }
     
+    public static void registerCraftTweakerBracketHandlers(final BiConsumer<String, BracketExpressionParser> reg) {
```

Comment:
Do we really want a consumer here, or another FU (with the same semantics) that describes what the two types stand for?